operations/agent_monitor.go

```diff
@@ -118,6 +121,11 @@ func agentMonitor() cli.Command {
 				Value: "bash",
 				Usage: "the path to the shell for starting the agent",
 			},
+			cli.StringFlag{
+				Name:  logOutputFlagName,
+				Value: string(agent.LogOutputFile),
+				Usage: "location for the agent monitor's log output (file, stdout)",
+			},
 			cli.StringFlag{
 				Name:  logPrefixFlagName,
 				Value: monitorLoggerName,
```

Comment:
should usage here also be switched to `log output` instead of `log name`?

Reply:
Not sure if you're talking about the prefix flag or the output flag, but the flag value/usage fields seems appropriate for the log prefix.

Reply:
Oh I see, yeah I can make them say the same thing.